### ISSUE TITLE
Adding a System label does not trigger a event based replication

### ISSUE THREAD

**Issue by shashank-k** [goharbor/harbor#13015]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**

Once you add a system label to an image, it should trigger a replication which is event based.

**Steps to reproduce the problem:**

- Ensure you have a replication rule set for replication
- Add a system label to the existing image. 
- This adding of the label should trigger the replication rule

**Versions:**
Please specify the versions of following systems.

- harbor version: Version v1.10.0-6b84b62f

**Comment by shashank-k** [comment]
Apologies for this for filing as a bug report. It is more of a feature request though.

**Comment by steven-zou** [comment]
@shashank-k 

Correct, it's a feature not implemented. It's in our roadmap. Please stay tuned.

**Comment by flbla** [comment]
@xaleeks @steven-zou : hi, any news about this feature ?

**Comment by Gui13** [comment]
Interested by this feature. For artifact promotion this is a nice to have feature (adding a label triggers a replication to a production repo).

**Comment by trivoallan** [comment]
Hi, could be very useful for artifact promotion indeed. At the moment we are using scheduled replication that scans all projects, it works but is not really efficient.
